### PR TITLE
OCPBUGS#9448: Add a note regarding checking the cluster version

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.9+

Issue: https://issues.redhat.com/browse/OCPBUGS-9448

Link to docs preview: https://57136--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-cluster-cli.html#update-upgrading-cli_updating-cluster-cli

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by lahinson — COMMENTED** [review]
I suggested a slight revision for clarity, but otherwise, LGTM!

File: modules/update-upgrading-cli.adoc [hunk 1]
```diff
@@ -122,6 +122,15 @@ Channel: stable-4.10 (available channels: candidate-4.10, candidate-4.11, eus-4.
 No updates available. You may force an upgrade to a specific release image, but doing so might not be supported and might result in downtime or data loss.
 ----
 +
+[NOTE]
+====
+You can ignore if the `oc get clusterversion` command gives the following error while `PROGRESSING` is `True`:
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/update-upgrading-cli.adoc]: Addressed

**Comment by jboxman-rh** [comment]
@xenolinux, looks like this doesn't merge cleanly into 4.9; Is this something that you can fix? Thanks!

**Comment by xenolinux** [comment]
> @xenolinux, looks like this doesn't merge cleanly into 4.9; Is this something that you can fix? Thanks!

Yes. Opened a manual CP PR https://github.com/openshift/openshift-docs/pull/57454

**Pull request merged**